src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveNode.java

```diff
@@ -1515,7 +1522,7 @@ private synchronized void setSleepTimer() {
         if (isInitializationComplete()) {
             timerDelay = sleepDelay;
         } else {
-            timerDelay = 5000;
```

Comment:
Thanks for the explanation, makes sense. I will close my PR.

Reply:
Totally unrelated-  Thanks for the CT101 humidity quirk fix. Been using for over a year!

Mildly related - 30 seconds for your device to initialize seems long. During my testing I had a logger.info between the one second iterations to see what step the initialization was on and the count. 18 seconds for me was at least two sigma, so that's how I arrived at that figure. (I started at 25).

Bob